### ISSUE TITLE
Support strike-through formatting

### ISSUE THREAD

**Issue by earshinov** [telegramdesktop/tdesktop#5079]
As suggested in https://github.com/telegramdesktop/tdesktop/issues/330#issuecomment-326881955, I am opening a separate ticket for a formatting feature that remained unimplemented after #330.

### Expected behavior

I would expect text entered like `-text-` or `~text~` to appear strike-through.

### Actual behavior

The entered text appears as is.

### Configuration
**Operating system:** Windows 10

**Version of Telegram Desktop:** 1.3.10

**Comment by earshinov** [comment]
@stek29 , Is there a ticket for strike-through support in the API? If not, I guess it should be created and linked in this ticket. I am ready to do it if you direct me to the API issue tracker.

**Comment by stek29** [comment]
@earshinov there's no open issue tracker for api issues, sorry. Feel free to contact Telegram via inapp support or on Twitter.

**Comment by earshinov** [comment]
Just found out that the current Telegram Desktop version supports strike-through formatting with `~~text~~`
